np-gravity.py

```diff
@@ -4,39 +4,40 @@
 
 import argparse
 import pygame
-from puzzle import Puzzle, DEFAULT_3SAT
+# from pathlib import Path
```

Comment:
This would be useful for platform-independent file accesses (e.g. handling how Windows and Mac/Linux use different types of slashes in paths). I wasn't sure if it were actually necessary, so try running the program and let me know if it breaks. If so, I can use this instead of a raw path.